src/components/rangeSelect.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from "react"
+
+import Select from "./select"
+
+interface Props {
+  name: {
```

Comment:
I'm not sure if this would be nicer but since you have that many props for min/max maybe we could have 1 object defining everything for min and another one defining everything for max. What do you think?

Reply:
Or as @Averethel mentioned, define only one name, value and so on and use suffix for min and max